app/models/get_involved.rb

```diff
@@ -0,0 +1,48 @@
+class GetInvolved < ContentItem
+  attr_reader :take_part_pages
+
+  def initialize(content_store_response, override_content_store_hash: nil)
+    super(content_store_response, override_content_store_hash:)
```

Comment:
For the sake of simplicity, if you call super without any parameters it takes the parameters from the calling method. 

So you could replace `super(content_store_response, override_content_store_hash:)` with just `super`